### PR TITLE
Write specific fields only

### PR DESCRIPTION
**Pull request by matt-frey**
This PR
* changes the writing of fields: Fields can be enabled and disabled.
* reduces the amount of information printed in verbose mode

### CODE REVIEW

File: src/3d/parcels/parcel_container.f90 [hunk 1]
```diff
@@ -231,7 +231,7 @@ end function get_dely
         subroutine parcel_replace(n, m)
             integer, intent(in) :: n, m
 
-#ifdef ENABLE_VERBOSE
+#if defined (ENABLE_VERBOSE) && !defined (NDEBUG)
```
Comment (sjboeing) [inline src/3d/parcels/parcel_container.f90:234]: Can you let me know why this combination is needed now?

Comment (matt-frey) [reply src/3d/parcels/parcel_container.f90:234]: I just wanted to reduce the amount of info printed to the standard output in verbose mode. This output is mainly useful for debugging.

**Review by sjboeing — APPROVED** [review]
I just had a quick look, it looks good. I just left one comment. Happy for you to merge when tested on some cases (e.g. beltrami and moist, that should cover most situations). Left one comment where I just wasn't quite sure of the logic.

**Comment by matt-frey** [comment]
I have already tested for Beltrami. All good.

**Pull request merged**